.vscode/settings.json

```diff
@@ -25,5 +25,6 @@
       }
     },
   ],
-  "java.test.defaultConfig": "WPIlibUnitTests"
+  "java.test.defaultConfig": "WPIlibUnitTests",
+  "java.format.settings.url": ".vscode/eclipse-java-google-style.xml"
```

Comment:
this should be set to `.github/sun_checks.xml`

Reply:
@WindowsVistaisCool You need to change the online version then, because this is the fix for redhat formatter